### PR TITLE
feat: add T&C link to init script

### PR DESCRIPTION
**Pull request by andyantrim**

### CODE REVIEW

File: nyx_client/cli/init_env.py [hunk 1]
```diff
@@ -35,6 +35,15 @@ def init_env(filename: str = ".env"):
             click.echo("Exiting with no changes")
             return
 
+    # Check user is aware of T&Cs
+    tcs = input(
+        "The Nyx SDK allows the sharing of links through a Nyx Instance. "
```
Comment (conbon) [inline nyx_client/cli/init_env.py:40]: Is this enough? or do we want to call out each point as in UI?

Comment (andyantrim) [reply nyx_client/cli/init_env.py:40]: Valid point, I think copying the UI makes sense, even if it's just for consistence

Comment (andyantrim) [reply nyx_client/cli/init_env.py]: https://github.com/Iotic-Labs/nyx-sdk/pull/20/commits/81377cffa14091585ae573bccbbe71f62609d504

File: nyx_client/cli/init_env.py [hunk 2]
```diff
@@ -21,6 +21,19 @@
...
+TERMS = """
+To use Nyx you must agree to our Terms of service when sharing content
+
+
+I understand that the data I am uploading will be visible in the Nyx Playground,
+including to users outside my organisation.
+I confirm that I have the right to share this data.
+I confirm that this data does not contain any Personally Identifiable Information
+or otherwise sensitive information, and that it does not violate any laws.
+I confirm I have read the Nyx Terms of Service and I am content to proceed.
```
Comment (conbon) [inline nyx_client/cli/init_env.py:33]: link?

File: nyx_client/cli/init_env.py [hunk 3]
```diff
@@ -21,6 +21,19 @@
 from nyx_client.cli.common import SDK_CLI_DEFAULT_HEADERS
 from nyx_client.identity_auth import IdentityAuth
 
+TERMS = """
+To use Nyx you must agree to our Terms of service when sharing content
```
Comment (conbon) [inline nyx_client/cli/init_env.py:25]: Capitalise Service?

File: nyx_client/cli/init_env.py [hunk 4]
```diff
@@ -35,6 +48,12 @@ def init_env(filename: str = ".env"):
             click.echo("Exiting with no changes")
             return
 
+    # Check user is aware of T&Cs
+    tcs = input(TERMS)
+    if tcs.lower() != "y":
+        click.echo("You must agree to the terms of service!")
```
Comment (conbon) [inline nyx_client/cli/init_env.py:54]: `Terms of Service`?

**Review by conbon — APPROVED** [review]

**Pull request merged**